Tests/SQLTests/DataQueryTests.swift

```diff
@@ -62,10 +62,11 @@ final class DataQueryTests: XCTestCase {
 
         let column = DataComputedColumn(function: "YEAR", columns: [.init(table: "foo", name: "date")])
         select.groupBys = [.computed(column)]
+        select.orderBys = [DataOrderBy(columns: [DataColumn(table: "foo", name: "name")], direction: .descending)]
```

Comment:
can we add a sort to this test as well to make sure the ordering is correct going forward :)

Reply:
I am confused? you just commented on it @tanner0101? Or would you like a separate test only to check that?